packages/mdl-menu/simple/mdl-simple-menu.scss

```diff
@@ -0,0 +1,187 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "mdl-animation/variables";
+@import "mdl-elevation/mixins";
+@import "mdl-theme/mixins";
+@import "mdl-typography/mixins";
+
+$mdl-simple-menu-fade-duration: .2s;
+$mdl-simple-menu-item-fade-duration: .3s;
+
+// Move text slightly when opening the menu, to improve visual flow.
+$mdl-simple-menu-item-offset: 10px;
+
+/* postcss-bem-linter: define simple-menu */
+.mdl-simple-menu {
+  display: none;
+  position: absolute;
+  max-width: calc(100vw - 32px);
+  max-height: calc(100vh - 32px);
+  margin: 0;
+  padding: 0;
+  transform: scale(0);
+  transform-origin: top left;
+  border-radius: 2px;
+  background-color: white;
+  opacity: 0;
+  overflow: hidden;
+  box-sizing: border-box;
+  will-change: transform, opacity;
+
+  @include mdl-elevation(2);
+
+  @include mdl-theme-dark(".mdl-simple-menu") {
+    background-color: #424242;
+  }
+
+  &--open {
+    display: inline-block;
+    transform: scale(1);
+    opacity: 1;
+  }
+
+  &--animating {
+    display: inline-block;
+    transition: opacity $mdl-simple-menu-fade-duration $mdl-animation-linear-out-slow-in-timing-function;
+  }
+
+  &__items {
+    display: inline-block;
+    transform-origin: $mdl-simple-menu-item-offset $mdl-simple-menu-item-offset;
+    overflow-x: hidden;
+    overflow-y: auto;
+    box-sizing: border-box;
+    will-change: transform;
+
+    /* stylelint-disable plugin/selector-bem-pattern, selector-no-universal */
+    > * {
```

Comment:
Hmm, you _do_ style `.mdl-list` specific stuff below. I might be inclined to limit the scope of the styles to just that, but I also don't see anything particularly wrong with this.